WalletWasabi/WabiSabi/Backend/Rounds/RoundParameters.cs

```diff
@@ -19,15 +19,19 @@ public class RoundParameters
 			Random = random;
 			FeeRate = feeRate;
 
-			MaxInputCountByAlice = wabiSabiConfig.MaxInputCountByAlice;
+			MaxInputCountByRound = wabiSabiConfig.MaxInputCountByRound;
```

Comment:
Note that max input count is redundant with multiparty transaction parameters' max vsize and vsize per alice parameters.

There are 3 variables but 2 degrees of freedom, so IMO we should use only 2 variables in the code (i prefer vsize based, and removing max input count vars), including the configuration, and eliminate one of the redundant concepts. We can still derive an effective value if necessary for client UI purposes.

Reply:
I would prefer to do this in a separate PR because the number of files we need to touch to remove this concept is big.